pkg/scheduler/apis/config/v1beta2/zz_generated.conversion.go

```diff
@@ -761,6 +761,7 @@ func Convert_v1beta2_Plugins_To_config_Plugins(in *v1beta2.Plugins, out *config.
 }
 
 func autoConvert_config_Plugins_To_v1beta2_Plugins(in *config.Plugins, out *v1beta2.Plugins, s conversion.Scope) error {
+	// WARNING: in.Enqueue requires manual conversion: does not exist in peer-type
```

Comment:
flagging this to make sure it was intentional this field was only added to the v1 type. The release note should be clear about what field is added and to which versions, and/or link to docs for the feature config

Reply:
Adding it to previous existing versions should be ok. @Huang-Wei wdyt?

Reply:
you don't have to, if you want to use this as a carrot to incentivize upgrading to v1 config, that can be fine. I just wanted to make sure it was an intentional decision

Reply:
though I would ask, if someone uses a v1beta1 default config, is the behavior different than a v1 default config?

Reply:
yes if the schedulingGates feature flag is enabled, so we have to add it.

Reply:
ok, let me add it to all supported versions.